testing/regress/ecl/key/phonenumber.xml

```diff
@@ -0,0 +1,328 @@
+ <Row><phonenumber>+44 20 7946 0958</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>UK international number without country context</description><number>+442079460958</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>GB</regioncode><countrycode>44</countrycode></Row>
+ <Row><phonenumber>+49 30 123456</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>German international number without country context</description><number>+4930123456</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>DE</regioncode><countrycode>49</countrycode></Row>
+ <Row><phonenumber>+81 3 1234 5678</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>Japanese international number without country context</description><number>+81312345678</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>JP</regioncode><countrycode>81</countrycode></Row>
+ <Row><phonenumber>+86 10 1234 5678</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>Chinese international number without country context</description><number>+861012345678</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>CN</regioncode><countrycode>86</countrycode></Row>
+ <Row><phonenumber>+33 1 42 68 53 00</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>French international number without country context</description><number>+33142685300</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>FR</regioncode><countrycode>33</countrycode></Row>
+ <Row><phonenumber>+39 06 698 80000</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>Italian international number without country context</description><number>+390669880000</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>VA</regioncode><countrycode>39</countrycode></Row>
+ <Row><phonenumber>+1 800 555 1212</phonenumber><country_code></country_code><tr_valid>true</tr_valid><description>US toll-free without country context</description><number>+18005551212</number><error>0</error><valid>true</valid><linetype>3</linetype><regioncode>US</regioncode><countrycode>1</countrycode></Row>
+ <Row><phonenumber>+49 30 123</phonenumber><country_code></country_code><tr_valid>false</tr_valid><description>Invalid German international without country context</description><number>+4930123</number><error>0</error><valid>true</valid><linetype>0</linetype><regioncode>DE</regioncode><countrycode>49</countrycode></Row>
+</Dataset>
+<Dataset name='invalidNumbers'>
+ <Row><phonenumber>+1 555 012 3456</phonenumber><country_code>US</country_code><tr_valid>false</tr_valid><description>US with international prefix</description><number>+15550123456</number><error>0</error><valid>false</valid><linetype>11</linetype><regioncode>ZZ</regioncode><countrycode>1</countrycode></Row>
```

Comment:
Why is this result invalid?  There are a few of these which have error==0 but are invalid?